app/src/main/java/com/myfitnesstracker/dto/BMI.kt

```diff
@@ -1,4 +1,4 @@
-package com.myfitnesstracker.ui.dto
```

Comment:
The import didn't need UI

Reply:
Yes... looking at the directory, there is no /ui/ directory, and the package name should match the directory.

Also, you normally would not have a DTO package inside of a UI package... these are separate concepts.